opportunities.go

```diff
@@ -0,0 +1,109 @@
+package chartmogul
+
+type Opportunity struct {
+	UUID string `json:"uuid"`
+	// Basic info
+	CustomerUUID       string `json:"customer_uuid"`
+	Owner              string `json:"owner"`
+	Pipeline           string `json:"pipeline"`
+	PipelineStage      string `json:"pipeline_stage"`
+	EstimatedCloseDate string `json:"estimated_close_date"`
+	Currency           string `json:"currency"`
+	AmountInCents      int    `json:"amount_in_cents"`
+	Type               string `json:"type"`
+	ForecastCategory   string `json:"forecast_category"`
+	WinLikelihood      int    `json:"win_likelihood"`
+	Custom             map[string]interface{}  `json:"custom,omitempty"`
+	CreatedAt          string `json:"created_at"`
+	UpdatedAt          string `json:"updated_at"`
+}
+
+type UpdateOpportunity struct {
+	Owner              string `json:"owner,omitempty"`
+	Pipeline           string `json:"pipeline,omitempty"`
+	PipelineStage      string `json:"pipeline_stage,omitempty"`
+	EstimatedCloseDate string `json:"estimated_close_date,omitempty"`
+	Currency           string `json:"currency,omitempty"`
+	AmountInCents      int    `json:"amount_in_cents,omitempty"`
+	Type               string `json:"type,omitempty"`
+	ForecastCategory   string `json:"forecast_category,omitempty"`
+	WinLikelihood      int    `json:"win_likelihood,omitempty"`
+	Custom             []Custom `json:"custom,omitempty"`
+}
+
+type NewOpportunity struct {
+	// Obligatory
+	CustomerUUID       string `json:"customer_uuid"`
+	Owner              string `json:"owner"`
+	Pipeline           string `json:"pipeline"`
+	PipelineStage      string `json:"pipeline_stage"`
+	EstimatedCloseDate string `json:"estimated_close_date"`
+	Currency           string `json:"currency"`
+	AmountInCents      int    `json:"amount_in_cents"`
+
+	//Optional
+	Type               string `json:"type,omitempty"`
+	ForecastCategory   string `json:"forecast_category,omitempty"`
+	WinLikelihood      int    `json:"win_likelihood,omitempty"`
+	Custom             []Custom `json:"custom,omitempty"`
+}
+
+// ListOpportunitiesParams = parameters for listing customer opportunities in API.
+type ListOpportunitiesParams struct {
+	CustomerUUID string `json:"customer_uuid"`
+	Cursor
+}
+
+// Opportunities is result of listing opportunities in API.
+type Opportunities struct {
+	Entries []*Opportunity `json:"entries"`
+	Pagination
+}
+
+const (
+	singleOpportunityEndpoint = "opportunities/:uuid"
+	opportunitiesEndpoint      = "opportunities"
+)
```

Comment:
[minor] we can align two variables

```suggestion
const (
	singleOpportunityEndpoint = "opportunities/:uuid"
	opportunitiesEndpoint     = "opportunities"
)
```

Reply:
thnx, applied 👍